dashboard/.jshintrc

```diff
@@ -4,15 +4,17 @@
   "browserify": true,
   "camelcase": true,
   "curly": true,
+  "devel": true,
   "eqeqeq": true,
   "forin": true,
   "freeze": true,
   "globals": {
-    "$": false
+    "angular": false
   },
   "immed": true,
   "indent": 2,
-  "latedef": true,
+  "jquery": true,
+  "latedef": "nofunc",
```

Comment:
@christianvuerings Please check my explanation below. The relevant portion is discussed here: https://github.com/johnpapa/angular-styleguide#bindable-members-up-top